src/cpp/ext/otel/otel_plugin.h

```diff
@@ -132,6 +133,12 @@ class OpenTelemetryPluginBuilder {
   OpenTelemetryPluginBuilder& SetLabelsInjector(
       std::unique_ptr<LabelsInjector> labels_injector);
 
+  // If set, \a channel_selector is called once per channel to decide whether to
+  // collect metrics on the channel or not.
+  OpenTelemetryPluginBuilder& SetChannelSelector(
```

Comment:
Given our discussion about eventually wanting to separate the target and channel scopes, it seems like this is actually the target-level scope selector, not the channel-level scope selector.  We may eventually want a channel-level selector that takes channel args as input.  I don't want to actually add that right now, but let's leave ourselves room to do that -- can we rename this to `SetTargetSelector()`?

Reply:
yeah, i'm not gonna make this part of the API in the public header, but yeah I'll make the name change anyway.